packages/pg-pubsub/src/index.ts

```diff
@@ -56,6 +57,11 @@ const plugin: PostGraphilePlugin = {
 
   ["postgraphile:options"](incomingOptions, { pgPool }) {
     const eventEmitter = new EventEmitter();
+    if (incomingOptions.subscriptionEventEmitterMaxListeners) {
```

Comment:
We want to allow `0` here, I think?

```suggestion
    if (incomingOptions.subscriptionEventEmitterMaxListeners != null) {
```

Reply:
Yes, indeed.